CHANGELOG.rst

```diff
@@ -1,6 +1,17 @@
 Change log
 ==========
 
+0.19.19 (2023-11-22)
+--------------------
+
+Bug fixes
+~~~~~~~~~
+
+* **report:** support originalUriBaseIds fields for SARIF
+  (causing malfunction of underline comments to the Swarm)
+* **doc:** update RTD config format, causing doc builds to fail
```

Comment:
So you updated format (?) of the config to make it fail?...
BTW, probably most users will not understand what "doc builds" you mean here.

Reply:
Should I rephrase it like "making RTD docs available again", or simply remove this line completely?

Reply:
Rephrase that readthedocs (full link) builds were not working.
And you didn't update the format, you fixed config to conform to the new format introduced by rtd.